Source/Stationeers.Addons/Modules/HarmonyLib/HarmonyModule.cs

```diff
@@ -25,6 +28,14 @@ public void Initialize()
         /// <inheritdoc />
         public IEnumerator Load()
         {
+            Debug.Log("Patching WorkshopManager.PublishWorkshop using Harmony...");
+            MethodInfo publishWorkshopMethod = typeof(WorkshopManager).GetMethod("PublishWorkshop", BindingFlags.Public | BindingFlags.Instance);
+            MethodInfo onCreateItemMethod = typeof(WorkshopManager).GetMethod("OnCreateItem", BindingFlags.NonPublic | BindingFlags.Instance);
+            MethodInfo publishWorkshopPrefixMethod = typeof(WorkshopManagerPatch).GetMethod("PublishWorkshopPrefix");
+            MethodInfo onCreateItemPostfixMethod = typeof(WorkshopManagerPatch).GetMethod("OnCreateItemPostfix");
+            _harmony.Patch(publishWorkshopMethod, new HarmonyMethod(publishWorkshopPrefixMethod));
+            _harmony.Patch(onCreateItemMethod, null, new HarmonyMethod(onCreateItemPostfixMethod));
```

Comment:
In case of a critical game change, I'd suggest to wrap this reflection code into try/catch, and display an error using `AlertPanel.Instance.ShowAlert($"Failed to initialize workshop publish patch!\n", AlertState.Alert);`. 

This might prevent someone from publishing addon with unwanted files once it crashes and the person does not notice it.